common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val WORKER_RPC_PORT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.rpc.port")
+      .withAlternative("rss.worker.rpc.port")
+      .categories("worker")
+      .doc("")
+      .version("0.2.0")
+      .intConf
+      .createWithDefault(0)
+
+  val WORKER_PUSH_SERVER_PORT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.pushServer.port")
```

Comment:
how about eliminating "server"? "celeborn.worker.push.port" should be clear enough, and consistent w/ "celeborn.worker.rpc.port"

Reply:
Done